### PR TITLE
[8.0] Add survey_partner_tag_share module

### PR DESCRIPTION
**Pull request by Ehtaga**
This module allows to share a survey to all the partners associated to a given tag.

The recipients are now a readonly field.
If you add a partner, it will be added to the recipients.
If you add a tag, all the partners who have this tag will be added to the recipients.

The recipients field doesn't contain any duplicate, and it is sorted by display_name.

### CODE REVIEW

**Comment by rafaelbn** [comment]
Please @JavierIniesta @JMODaERP review, thanks!

File: survey_partner_tag_share/README.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Survey Partner Tag Share
+========================
+
+This module was written to extend the functionality of sharing a survey. It
```
Comment (elicoidal) [inline survey_partner_tag_share/README.rst]: This module extends

File: survey_partner_tag_share/README.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Create a survey
+* Give a tag to some of your partners
+* Go on your survey page, share and invite
+* Choose a method which send emails
```
Comment (elicoidal) [inline survey_partner_tag_share/README.rst]: Select the method to send emails

File: survey_partner_tag_share/README.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (elicoidal) [inline survey_partner_tag_share/README.rst]: https

File: survey_partner_tag_share/__init__.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline survey_partner_tag_share/__init__.py:1]: Please use simple headers following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

File: survey_partner_tag_share/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Survey Partner Tag Share',
```
Comment (elicoidal) [inline survey_partner_tag_share/__openerp__.py:5]: Please follow the guidelines from:
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
Version 8.0.1.0.0
Remove empty keys
Remove maintainer key

**Comment by gurneyalex** [comment]
:+1:

**Comment by rafaelbn** [comment]
Hi @Ehtaga if you could review @elicoidal comments I will review too and get this merged! :smile:

**Comment by Ehtaga** [comment]
@rafaelbn @elicoidal The README and Manifest files have been updated

File: survey_partner_tag_share/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright <YEAR(S)> <AUTHOR(S)>
```
Comment (rafaelbn) [inline survey_partner_tag_share/__openerp__.py:2]: @Ehtaga authors are missing

**Comment by rafaelbn** [comment]
Tested in runbot :+1: 
Just must be fixed authors, thanks!

**Comment by rafaelbn** [comment]
@Ehtaga just remains authors to merge this! :smile:

**Comment by gurneyalex** [comment]
I just manually fixed the boilerplate code and merged.

**Pull request merged**